_includes/bot-integration-data.html

```diff
@@ -1,20 +1,18 @@
-<div class="preview" style="position: absolute; display: none; width: 0px;">
-   <div class="preview-page">
+<div class="preview" style="position: absolute; display: none; width: 0px">
```

Comment:
Are you open to adjusting one more thing here? (Otherwise, let’s do it separately.)

Is this element shown in the first place, is something overriding `display: none`?

If yes, can we peg these styles to the `preview` classes? Or, can we create another class for this?

Then, please remove “px” here—not needed for 0 values. (Let’s check our CSS on more of those occurrences. Will take note, too.)

Reply:
let's open a different branch for this, let me please focus on the bubble for this pr